### PR TITLE
[apps/ca] Useless nullity test

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
Very short change despite the diff length.

##### Checklist
- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by levitte** [comment]
I've labeled this for 1.1.0 as well.

File: apps/ca.c [hunk 1]
```diff
@@ -442,30 +442,27 @@ int ca_main(int argc, char **argv)
...
-                BIO_free(oid_bio);
-            }
-        }
-        if (!add_oid_section(conf)) {
+        if (oid_bio == NULL) {
+            /*-
+            BIO_printf(bio_err,"problems opening %s for extra oid's\n",p);
             ERR_print_errors(bio_err);
-            goto end;
+            */
```
Comment (levitte) [inline apps/ca.c]: Feel free to remove the dead code as well.

Comment (FdaSilvaYY) [reply apps/ca.c]: Working on it, that's why I see this ;)

**Comment by FdaSilvaYY** [comment]
Ping @levitte : PR ready even if label is missing ... ;)

**Comment by levitte** [comment]
Merged.

master: df364297495e1d7409f1fcb0f4432c8671019cd8
1.1.0: b2621ffdf8063c961ee5c3a7dd9db68d623d3554

**Pull request closed without merging**